### PR TITLE
[9.101.x] NO-ISSUE: fix sonataflow management console ocp permission denied

### PR DESCRIPTION
**Pull request by fantonangeli**
Fix /var/www/html permissions for OCP platform

**Note:** this PR still need to be tested

### CODE REVIEW

File: packages/sonataflow-management-console-image/resources/modules/sonataflow/management-console/configure-httpd.sh [hunk 1]
```diff
@@ -44,4 +44,8 @@ cp -v "${SCRIPT_DIR}"/added/* "${MGMT_CONSOLE_HOME}"/launch
 chmod +x "${MGMT_CONSOLE_HOME}/launch/entrypoint.sh" "${MGMT_CONSOLE_HOME}/image-env-to-json-standalone"
 chown -R "${USER_ID}" "${MGMT_CONSOLE_HOME}"
 
+# Fixing /var/www permissions
+chgrp -R 0 /var/log/httpd /var/run/httpd /var/www/html /management-console \
+chmod -R g=u /var/log/httpd /var/run/httpd /var/www/html /management-console \
```
Comment (ricardozanini) [inline packages/sonataflow-management-console-image/resources/modules/sonataflow/management-console/configure-httpd.sh:49]: Please use the env vars:

```
ENV HTTPD_CONTAINER_SCRIPTS_PATH=/usr/share/container-scripts/httpd/ \
    HTTPD_APP_ROOT=${APP_ROOT} \
    HTTPD_CONFIGURATION_PATH=${APP_ROOT}/etc/httpd.d \
    HTTPD_MAIN_CONF_PATH=/etc/httpd/conf \
    HTTPD_MAIN_CONF_MODULES_D_PATH=/etc/httpd/conf.modules.d \
    HTTPD_MAIN_CONF_D_PATH=/etc/httpd/conf.d \
    HTTPD_TLS_CERT_PATH=/etc/httpd/tls \
    HTTPD_VAR_RUN=/var/run/httpd \
    HTTPD_DATA_PATH=/var/www \
    HTTPD_DATA_ORIG_PATH=/var/www \
    HTTPD_LOG_PATH=/var/log/httpd
```

File: packages/osl-management-console-image/generated/modules/osl/management-console/configure-httpd.sh [hunk 2]
```diff
@@ -44,4 +44,20 @@ cp -v "${SCRIPT_DIR}"/added/* "${MGMT_CONSOLE_HOME}"/launch
...
+    HTTPD_MAIN_CONF_D_PATH=/etc/httpd/conf.d \
+    HTTPD_TLS_CERT_PATH=/etc/httpd/tls \
+    HTTPD_VAR_RUN=/var/run/httpd \
+    HTTPD_DATA_PATH=/var/www \
+    HTTPD_DATA_ORIG_PATH=/var/www \
+    HTTPD_LOG_PATH=/var/log/httpd
+
+# Fixing /var/www permissions
+chgrp -R 0 /var/log/httpd /var/run/httpd /var/www/html /management-console \
+chmod -R g=u /var/log/httpd /var/run/httpd /var/www/html /management-console \
```
Comment (ricardozanini) [inline packages/osl-management-console-image/generated/modules/osl/management-console/configure-httpd.sh:61]:
```suggestion
chgrp -R 0 ${HTTPD_LOG_PATH} ${HTTPD_VAR_RUN} ${HTTPD_DATA_PATH}/html /management-console \
chmod -R g=u ${HTTPD_LOG_PATH} ${HTTPD_VAR_RUN} ${HTTPD_DATA_PATH}/html /management-console \
```

File: packages/osl-management-console-image/generated/modules/osl/management-console/configure-httpd.sh [hunk 3]
```diff
@@ -44,4 +44,20 @@ cp -v "${SCRIPT_DIR}"/added/* "${MGMT_CONSOLE_HOME}"/launch
...
+    HTTPD_APP_ROOT=${APP_ROOT} \
+    HTTPD_CONFIGURATION_PATH=${APP_ROOT}/etc/httpd.d \
+    HTTPD_MAIN_CONF_PATH=/etc/httpd/conf \
+    HTTPD_MAIN_CONF_MODULES_D_PATH=/etc/httpd/conf.modules.d \
+    HTTPD_MAIN_CONF_D_PATH=/etc/httpd/conf.d \
+    HTTPD_TLS_CERT_PATH=/etc/httpd/tls \
+    HTTPD_VAR_RUN=/var/run/httpd \
+    HTTPD_DATA_PATH=/var/www \
+    HTTPD_DATA_ORIG_PATH=/var/www \
+    HTTPD_LOG_PATH=/var/log/httpd
```
Comment (ricardozanini) [inline packages/osl-management-console-image/generated/modules/osl/management-console/configure-httpd.sh:57]:
```suggestion

```

File: packages/sonataflow-management-console-image/resources/modules/sonataflow/management-console/configure-httpd.sh [hunk 4]
```diff
@@ -44,4 +44,20 @@ cp -v "${SCRIPT_DIR}"/added/* "${MGMT_CONSOLE_HOME}"/launch
...
+    HTTPD_MAIN_CONF_D_PATH=/etc/httpd/conf.d \
+    HTTPD_TLS_CERT_PATH=/etc/httpd/tls \
+    HTTPD_VAR_RUN=/var/run/httpd \
+    HTTPD_DATA_PATH=/var/www \
+    HTTPD_DATA_ORIG_PATH=/var/www \
+    HTTPD_LOG_PATH=/var/log/httpd
+
+# Fixing /var/www permissions
+chgrp -R 0 /var/log/httpd /var/run/httpd /var/www/html /management-console \
+chmod -R g=u /var/log/httpd /var/run/httpd /var/www/html /management-console \
```
Comment (ricardozanini) [inline packages/sonataflow-management-console-image/resources/modules/sonataflow/management-console/configure-httpd.sh:61]:
```suggestion
chgrp -R 0 ${HTTPD_LOG_PATH} ${HTTPD_VAR_RUN} ${HTTPD_DATA_PATH}/html /management-console \
chmod -R g=u ${HTTPD_LOG_PATH} ${HTTPD_VAR_RUN} ${HTTPD_DATA_PATH}/html /management-console \
```

File: packages/sonataflow-management-console-image/resources/modules/sonataflow/management-console/configure-httpd.sh [hunk 5]
```diff
@@ -44,4 +44,20 @@ cp -v "${SCRIPT_DIR}"/added/* "${MGMT_CONSOLE_HOME}"/launch
...
+    HTTPD_APP_ROOT=${APP_ROOT} \
+    HTTPD_CONFIGURATION_PATH=${APP_ROOT}/etc/httpd.d \
+    HTTPD_MAIN_CONF_PATH=/etc/httpd/conf \
+    HTTPD_MAIN_CONF_MODULES_D_PATH=/etc/httpd/conf.modules.d \
+    HTTPD_MAIN_CONF_D_PATH=/etc/httpd/conf.d \
+    HTTPD_TLS_CERT_PATH=/etc/httpd/tls \
+    HTTPD_VAR_RUN=/var/run/httpd \
+    HTTPD_DATA_PATH=/var/www \
+    HTTPD_DATA_ORIG_PATH=/var/www \
+    HTTPD_LOG_PATH=/var/log/httpd
```
Comment (ricardozanini) [inline packages/sonataflow-management-console-image/resources/modules/sonataflow/management-console/configure-httpd.sh:57]:
```suggestion

```

**Review by ricardozanini — APPROVED** [review]

File: packages/osl-management-console-image/generated/modules/osl/management-console/configure-httpd.sh [hunk 6]
```diff
@@ -44,4 +44,9 @@ cp -v "${SCRIPT_DIR}"/added/* "${MGMT_CONSOLE_HOME}"/launch
 chmod +x "${MGMT_CONSOLE_HOME}/launch/entrypoint.sh" "${MGMT_CONSOLE_HOME}/image-env-to-json-standalone"
 chown -R "${USER_ID}" "${MGMT_CONSOLE_HOME}"
 
+
+# Fixing /var/www permissions
+chgrp -R 0 ${HTTPD_LOG_PATH} ${HTTPD_VAR_RUN} ${HTTPD_DATA_PATH}/html /management-console \
```
Comment (rgdoliveira) [inline packages/osl-management-console-image/generated/modules/osl/management-console/configure-httpd.sh:49]: The generated is different from the sources, isn't it?
I guess you need need to update it and push again.

Comment (fantonangeli) [reply packages/osl-management-console-image/generated/modules/osl/management-console/configure-httpd.sh]: @rgdoliveira thank you, updated now

**Review by rgdoliveira — APPROVED** [review]

**Review by domhanak — APPROVED** [review]

**Pull request merged**